editoast/src/views/rolling_stock/towed.rs

```diff
@@ -0,0 +1,610 @@
+    pub gamma: Gamma,
+}
+
+impl From<TowedRollingStockForm> for Changeset<TowedRollingStockModel> {
+    fn from(towed_rolling_stock_form: TowedRollingStockForm) -> Self {
+        TowedRollingStockModel::changeset()
+            .railjson_version(ROLLING_STOCK_RAILJSON_VERSION.to_string())
+            .name(towed_rolling_stock_form.name)
+            .locked(towed_rolling_stock_form.locked)
+            .mass(towed_rolling_stock_form.mass)
+            .length(towed_rolling_stock_form.length)
+            .comfort_acceleration(towed_rolling_stock_form.comfort_acceleration)
+            .startup_acceleration(towed_rolling_stock_form.startup_acceleration)
+            .inertia_coefficient(towed_rolling_stock_form.inertia_coefficient)
+            .rolling_resistance(towed_rolling_stock_form.rolling_resistance)
+            .gamma(towed_rolling_stock_form.gamma)
+    }
+}
+
+/// Create a rolling stock
+#[utoipa::path(
+    post, path = "",
+    tag = "rolling_stock",
+    request_body = TowedRollingStockForm,
+    responses(
+        (status = 200, description = "The created towed rolling stock", body = TowedRollingStock)
+    )
+)]
+async fn post(
+    State(db_pool): State<DbConnectionPoolV2>,
+    Extension(authorizer): AuthorizerExt,
+    Json(towed_rolling_stock_form): Json<TowedRollingStockForm>,
+) -> Result<Json<TowedRollingStock>> {
+    let authorized = authorizer
+        .check_roles([BuiltinRole::RollingStockCollectionWrite].into())
+        .await
+        .map_err(AuthorizationError::AuthError)?;
+    if !authorized {
+        return Err(AuthorizationError::Unauthorized.into());
+    }
+    let conn = &mut db_pool.get().await?;
+    let rolling_stock_changeset: Changeset<TowedRollingStockModel> =
+        towed_rolling_stock_form.into();
+
+    let rolling_stock_model = rolling_stock_changeset.version(0).create(conn).await?;
+
+    Ok(Json(TowedRollingStock::from(rolling_stock_model)))
+}
+
+#[derive(Serialize, ToSchema)]
+#[cfg_attr(test, derive(Deserialize))]
+struct TowedRollingStockCountList {
+    #[schema(value_type = Vec<TowedRollingStock>)]
+    results: Vec<TowedRollingStock>,
+    #[serde(flatten)]
+    stats: PaginationStats,
+}
+
+#[utoipa::path(
+    get, path = "",
+    tag = "rolling_stock",
+    params(PaginationQueryParam),
+    responses(
+        (status = 200, body = inline(TowedRollingStockCountList)),
+    )
+)]
+async fn get_list(
+    State(db_pool): State<DbConnectionPoolV2>,
+    Extension(authorizer): AuthorizerExt,
+    Query(page_settings): Query<PaginationQueryParam>,
+) -> Result<Json<TowedRollingStockCountList>> {
+    let authorized = authorizer
+        .check_roles([BuiltinRole::RollingStockCollectionRead].into())
+        .await
+        .map_err(AuthorizationError::AuthError)?;
+    if !authorized {
+        return Err(AuthorizationError::Unauthorized.into());
+    }
+    let settings = page_settings
+        .validate(1000)?
+        .warn_page_size(100)
```

Comment:
```suggestion
        .validate(50)?
```

Since this is a new endpoint I'd suggest truly enforcing the pagination (50 seems like a good max to me, but feel free to change that). Wdyt?

Reply:
Good idea, I just copied/pasted but indeed. Done in 9e25169d761385f34419b82e843d4d08e3b8f8f8.